dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/createfolder/CreateFolderView.java

```diff
@@ -1,17 +1,14 @@
 package com.spectralogic.dsbrowser.gui.components.createfolder;
 
 import com.airhacks.afterburner.views.FXMLView;
-import com.spectralogic.dsbrowser.gui.DeepStorageBrowserPresenter;
 import com.spectralogic.dsbrowser.gui.util.StringConstants;
 
 public class CreateFolderView extends FXMLView {
-    public CreateFolderView(final CreateFolderModel createFolderModel, final DeepStorageBrowserPresenter deepStorageBrowserPresenter) {
+    public CreateFolderView(final CreateFolderModel createFolderModel) {
         super(name -> {
```

Comment:
As I understand it, any Presenter with an @ModelContext needs those model contexts available in the View.  This is how its done with similar CreateBucket(Presenter / View) and PhysicalPlacement(Presenter / View)

Are you suggesting we eliminate the default: return null;  ?

public class CreateFolderView extends FXMLView {
    public CreateFolderView(final CreateFolderModel createFolderModel) {
        return createFolderModel;
}

?